plugins/github-pull-requests-board/src/components/Card/Card.tsx

```diff
@@ -29,6 +29,7 @@ type Props = {
   isDraft: boolean;
   repositoryIsArchived: boolean;
   labels?: Label[];
+  status: Status;
```

Comment:
Wonder if it makes sense to make `status` optional, and only display it if it's passed for all the components?

Reply:
I think that's a good idea, I'll make the necessary changes

Reply:
As mentioned in #20546 - `being able to see which ones are already passing their checks without having to click into each one makes prioritizing them for review much faster.`
@benjdlambert I believe it's better to show all statuses regardless of whether it's passed.